docs/getting-started/security-ui.asciidoc

```diff
@@ -16,210 +17,110 @@ image::images/search-bar.png[]
-
-image::images/getting-started-pg.png[Shows the Get started page]
+TIP: Click the *Collapse side navigation* button to collapse and expand the main navigation menu.
 
-[float]
-[[overview-ui]]
-=== Overview dashboard
-
-The Overview dashboard provides a high-level snapshot of detections, external alerts, and event trends. It can help you assess overall system health and find anomalies that may require further investigation. Refer to <<overview-dashboard, Overview dashboard>> for more information.
-
-image::images/overview-pg.png[Overview dashboard]
+The {security-app} contains the following pages that enable analysts to view, analyze, and manage security data:
 
 [float]
-[[detection-response-dashboard-ui]]
-=== Detection & Response dashboard
+=== Dashboards
 
-The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts. Refer to <<detection-response-dashboard, Detection & Response dashboard>> for more information.
+Expand this section to access the Overview, Detection & Response, Kubernetes, and Cloud Posture dashboards, which provide interactive visualizations that summarize specific data within your environment. Refer to <<dashboards-overview>> for more information.
 
 [role="screenshot"]
-image::detections/images/detection-response-dashboard.png[Overview of Detection & Response dashboard]
+image::images/dashboards-pg.png[]
 
 [float]
-[[detection-engine-ui]]
-=== Alerts page
+=== Alerts
 
-The Alerts page allows you to view and manage all alerts to monitor activity within your network. Refer to <<detection-engine-overview, Detections and Alerts>> for more information.
+View and manage alerts to monitor activity within your network. Refer to <<detection-engine-overview>> for more information.
 
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
 [float]
-=== Findings page
+=== Findings
 
-The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
+Compare your Kubernetes infrastucture against a variety of security benchmarks. Refer to the <<findings-page>> to find out how to set this up.
 
 [role="screenshot"]
 image::cloud-native-security/images/findings-page.png[The Findings page]
 
 [float]
-[[rules-page]]
-=== Rules page
+=== Timelines
 
-The Rules page allows you to view and manage all detection rules. Refer to <<rules-ui-management, Manage detection rules>> for more information about prebuilt and custom rules.
+Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are interactive and allow you to share your findings with other team members. Refer to <<timelines-ui>> to learn more.
```

Comment:
```suggestion
Investigate alerts and complex threats such as lateral movement, across your network. Timelines are interactive and allow you to share your findings with other team members. Refer to <<timelines-ui>> to learn more.
```
This would also include threats other than malware